### PR TITLE
Update publishing setup for typesafe-ivy-releases

### PR DESCRIPTION
**Pull request by szeiger**
I've created a package in the repo and successfully staged a 1.1.0 release of this build at https://bintray.com/typesafe/ivy-releases/emoji in case you want to publish it (without merging #4 first).

### CODE REVIEW

**Comment by SethTisue** [comment]
(once this is merged, I'll need to fork the repo in the community build because (as usual) dbuild and bintray-sbt don't get along)

File: build.sbt [hunk 1]
```diff
@@ -14,11 +14,10 @@ scalaVersion := scala212Version
 
 crossScalaVersions := Seq(scala212Version, scala211Version, scala210Version)
 
-credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
-
-val typesafeIvyReleases = Resolver.url("typesafe-ivy-private-releases", new URL("http://private-repo.typesafe.com/typesafe/ivy-releases/"))(Resolver.ivyStylePatterns)
-
-publishTo := Some(typesafeIvyReleases)
+bintrayOrganization := Some("typesafe")
```
Comment (wsargent) [inline build.sbt]: shouldn't this be lightbend?

Comment (szeiger) [inline build.sbt]: The `ivy-releases` repo where the previous release was published (and then migrated to BinTray) is under the `typesafe` organization.

**Comment by SethTisue** [comment]
heartwarming to see the whole company pulling together on this

**Comment by SethTisue** [comment]
> I'll need to fork the repo in the community build

https://github.com/scala/community-builds/pull/519

**Pull request merged**